src/amazonqGumby/telemetry/codeTransformTelemetry.ts

```diff
@@ -44,17 +48,18 @@ export const logCodeTransformInitiatedMetric = (source: string): void => {
     }
 }
 
-//TODO: it would be better to expand JDKVersion from an enum to a class
-export const toJDKMetricValue = (source: JDKVersion): string => {
+export const JDKToTelemetryValue = (
```

Comment:
this does not match the function naming conventions of all other functions in the project.
```suggestion
export const jdkToTelemetryValue = (
```